VEHICLE/GetVehicleFlightNozzlePosition.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: VEHICLE
-aliases: ["0xDA62027C8BDB326E","_GET_PLANE_HOVER_MODE_PERCENTAGE","_GET_VEHICLE_HOVER_MODE_PERCENTAGE","_GET_PLANE_VTOL_DIRECTION"]
+aliases: ["0xDA62027C8BDB326E","_GET_PLANE_HOVER_MODE_PERCENTAGE","_GET_VEHICLE_HOVER_MODE_PERCENTAGE","_GET_PLANE_VTOL_DIRECTION", "_GET_VEHICLE_FLIGHT_NOZZLE_POSITION"]
```

Comment:
Considering it's the same name just without `_` prefix, I don't think the alias is needed here.